src/ITensors.jl

```diff
@@ -225,4 +225,14 @@ end
 #  _precompile_()
 #end
 
+
+
+#
+# See section on "Transition from normal dependency to extension"
+# in https://pkgdocs.julialang.org/v1/creating-packages
+#
+if !isdefined(Base, :get_extension)
+  include("../ext/ITensorsObserversExt.jl")
+end
+
```

Comment:
I question is the above code is really needed. When reading up on package extensions, I saw the section "Transition from normal dependency to extension" and thought since `Observers` is currently a normal dependency that maybe we need this?

Reply:
I don't think we need this. This would make `Observers` a conditional dependency in Julia v1.9 and above and a regular dependency in versions below v1.9. By using `PackageExtensionCompat` we can make it a conditional dependency for any version of Julia.

Reply:
Removed